packages/website/src/pages/feedback/Loading.tsx

```diff
@@ -12,6 +12,12 @@ const fullContent = `
     export default () => <Loading fullContent />
 `;
 
+const loadingSpinner = `
+    import {LoadingSpinner} from '@coveord/plasma-react';
+
+    export default () => <LoadingSpinner />
```

Comment:
```suggestion
    export default () => <div className="flex justify-center" ><LoadingSpinner /></div>
```

This should line it up, but I wonder if it would be worth it to imbed the `fullContent` prop to do this so that we can do `<loadingSpinner fullContent/>` to center

Reply:
I don't know where else this could be used but in it's current state, the component is really easy to position either in a new component or any place you want it really 🤔 

I also don't know if it will be ever used again BY UX outside of the Download toast (I've put it in the search bar without asking anyone 🙃 😇 )